### PR TITLE
Fix: eslint and prettier added

### PR DESCRIPTION
**Pull request by moiskillnadne**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,8 +5,10 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
-    "start": "tsc && node ./dist/src/app.js",
-    "test": "node --experimental-vm-modules node_modules/jest/bin/jest.js"
+    "start": "tsc && /Users/victorryabkov/.nvm/versions/node/v16.20.2/bin/node ./dist/src/app.js",
```
Comment (karser) [inline package.json:8]: remove /Users/victorryabkov/.nvm/versions/node/v16.20.2/bin/node

**Review by karser — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (moiskillnadne) [reply package.json:8]: Forgot about it. Thank you!

Comment (moiskillnadne) [reply package.json]: Done

**Review by karser — APPROVED** [review]

**Comment by moiskillnadne** [comment]
Wrong branch naming. Close this pr and reopen the same pr with branch name "fix-cleaning"

**Pull request closed without merging**